### ISSUE TITLE
CAPTCHA: Focus/hover state of the checkbox fails contrast ratio

### ISSUE THREAD

**Issue by sarasuri** [mmiron-HC/WST-RSAMS#167]
**Page:** All (CAPTCHA under Report a problem or mistake on this page)
**URL:** https://recalls-rappels.canada.ca/en
**User Agent:** Firefox (v99)
**Language:** English and French

**Issue Description:** The focus and hover states of the checkbox fail contrast ratio against its adjacent colour

Focus state = #4d90fe
Hover state = #b2b2b2
Adjacent colour= #f9f9f9

![image](https://user-images.githubusercontent.com/12256067/162526794-c4fe4c2e-e800-4ed5-88dc-0da71e1c6989.png)
![image](https://user-images.githubusercontent.com/12256067/162526928-d3b8948d-a7ee-48f3-821b-77b0ec03eb7c.png)

![image](https://user-images.githubusercontent.com/12256067/162526826-91c7fba2-c7b7-4fbc-92c4-f8a3ffacd7ce.png)
![image](https://user-images.githubusercontent.com/12256067/162526902-3af83cf6-93e9-4ae1-83b5-1bdbea4625ec.png)

**Steps to reproduce:**

1. Go to the above URL
2. Expand the Report a problem or mistake on this page component
3. Tab to the checkbox under CAPTCHA
4. Hover over the checkbox
5. Use Colour Contrast Analyser to verify the contrast ratio

**Expected results:** The focus and hover states of the checkbox  should have a contrast ratio of at least 3:1 against its adjacent colour.

**Comment by echouin** [comment]
believe this has been fixed.

**Comment by YuxuanXie01** [comment]
Retested, issue persists. 
Contrast ratio slightly falls below 3:1 when in focus state, where it should be at least 3:1 against adjacent colour: 
![image](https://github.com/user-attachments/assets/fa0cb8ae-8969-4c2d-9781-624efe0aa91c)



Contrast ratio remains at 2:1 when in hover state, where it should be at least 3:1 against adjacent colour:
![image](https://github.com/user-attachments/assets/6412c685-809c-426f-a8d9-8e2f6c6972a7)

**Comment by 40999221** [comment]
Issue is still exist. Focus state color contrast ratio is just below 3:1 as shown below:

![image](https://github.com/user-attachments/assets/26171e68-b91b-4c6d-bebd-8e010c72e5bd)

**Comment by zimw7** [comment]
Issue still exists. 
Focus state colour contrast ratio is below 3:1 as shown below:
![image](https://github.com/user-attachments/assets/263b5077-c990-43ac-848c-9b3270fca0ee)

Hover state colour contrast ratio is 2:1 as shown below:
![image](https://github.com/user-attachments/assets/bd5e3e4e-e10e-4ccd-b2cf-e75856e9ad87)

**Comment by mmiron-HC** [comment]
@YuxuanXie01 Please copy this ticket over to [ART-2605](https://jill.hc-sc.gc.ca/jira/browse/ART-2605) Put the status of this one to Escalated and please include a note to say that this is a google product limitation. Add product_limitation to the labels as well
